default/python/AI/AIstate.py

```diff
@@ -500,38 +529,65 @@ def update_system_status(self):
             sys_status['planetThreat'] = pattack * phealth
             sys_status['mydefenses'] = {'overall': mypattack * myphealth, 'attack': mypattack, 'health': myphealth}
 
-            if max(sys_status.get('totalThreat', 0), pattack * phealth) >= 0.6 * lost_fleet_rating:  # previous threat assessment could account for losses, ignore the losses now
+            # previous threat assessment could account for losses, ignore the losses now
+            if max(sys_status.get('totalThreat', 0), pattack * phealth) >= 0.6 * lost_fleet_rating:
                 lost_fleet_rating = 0
 
             # TODO use sitrep combat info rather than estimating stealthed enemies by fleets lost to them
             # TODO also only consider past stealthed fleet threat to still be present if the system is still obstructed
             # TODO: track visibility across turns in order to distinguish the blip of visibility in (losing) combat,
-            # which FO currently treats as being for the previous turn, partially superseding the previous visibility for that turn
-            if not partial_vis_turn == current_turn:  # (universe.getVisibility(sys_id, self.empire_id) >= fo.visibility.partial):
+            # which FO currently treats as being for the previous turn,
+            #  partially superseding the previous visibility for that turn
```

Comment:
Adjust line length and indent level to make clear all 3 lines belong to the same TODO